### PR TITLE
Set `FEED_EXPORT_ENCODING='utf-8'` in the default template

### LINKED ISSUE #5797: Set FEED_EXPORT_ENCODING="utf-8" in the default template
The default value for FEED_EXPORT_ENCODING (`None`) means `ensure_ascii=True` for JSON exports, because that was the existing behavior before that option was added. A better default is `"utf-8"`, which creates JSON files with `ensure_ascii=False`, so we decided to add this setting to the default template like we currently do with some other settings.

The setting doc should also get a note about this like in https://docs.scrapy.org/en/latest/topics/settings.html#twisted-reactor

### PR DESCRIPTION
**Pull request by Laerte**
Resolves #5797

### CODE REVIEW

File: tests/test_feedexport.py [hunk 1]
```diff
@@ -1298,8 +1298,8 @@ def test_export_encoding(self):
         items = [dict({'foo': 'Test\xd6'})]
 
         formats = {
-            'json': '[{"foo": "Test\\u00d6"}]'.encode('utf-8'),
-            'jsonlines': '{"foo": "Test\\u00d6"}\n'.encode('utf-8'),
+            'json': '[{"foo": "TestÖ"}]'.encode('utf-8'),
+            'jsonlines': '{"foo": "TestÖ"}\n'.encode('utf-8'),
```
Comment (Laerte) [inline tests/test_feedexport.py:1302]:
```suggestion
            'json': b'[{"foo": "Test\xc3\x96"}]',
            'jsonlines': b'{"foo": "Test\xc3\x96"}\n',
```
We can use this syntax if you guys prefer. :-)

**Comment by Laerte** [comment]
The only error is not related with the change, rerun should fix.

**Comment by wRAR** [comment]
The idea was to add it to [the project template](https://github.com/scrapy/scrapy/blob/master/scrapy/templates/project/module/settings.py.tmpl), not change the default :) Sorry if that wasn't clear.

**Comment by Laerte** [comment]
@wRAR Gotcha! sorry, check now!

File: docs/topics/feed-exports.rst [hunk 2]
```diff
@@ -515,6 +515,11 @@ which uses safe numeric encoding (``\uXXXX`` sequences) for historic reasons.
 
 Use ``utf-8`` if you want UTF-8 for JSON too.
 
+.. versionchanged:: 2.8
```
Comment (wRAR) [inline docs/topics/feed-exports.rst:518]: 2.8 here should be fine but please use `VERSION` next time to be more flexible.

File: docs/topics/feed-exports.rst [hunk 3]
```diff
@@ -515,6 +515,11 @@ which uses safe numeric encoding (``\uXXXX`` sequences) for historic reasons.
...
+   The :command:`startproject` command now sets this setting to
+   ``utf-8`` in the generated
+   ``settings.py`` file.
```
Comment (wRAR) [inline docs/topics/feed-exports.rst:521]: The line breaks here are weird :)

**Review by wRAR — APPROVED** [review]

File: docs/topics/feed-exports.rst (hunk 3 above)
Comment (Laerte) [reply docs/topics/feed-exports.rst:521]: Agreed! Fixed.

File: docs/topics/feed-exports.rst (hunk 2 above)
Comment (Laerte) [reply docs/topics/feed-exports.rst:518]:
```diff
-.. versionchanged:: 2.8
+.. versionchanged:: VERSION
```
Like this?

Comment (wRAR) [reply docs/topics/feed-exports.rst:518]: Yes, it will be replaced during the release process.

Comment (Laerte) [reply docs/topics/feed-exports.rst]: Cool, done as well.

**Review by Gallaecio — APPROVED** [review]

**Pull request merged**